### PR TITLE
Update issue templates to include more relevant information

### PR DESCRIPTION
**Pull request by pakrym**
Add a question about library name that would allow us to triage issues way easier, possibly automatically.

Change other questions to ask for information that's more useful for .net libraries.

Normalize the file names and make the templates shorter.

Drop the checklist, we don't use it for anything and people are free to remove it (which they do)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/question-query-template.md [hunk 1]
```diff
@@ -14,9 +14,9 @@ A clear and concise ask/query.
 A clear explanation of why is this not a bug or a feature request?
 
 **Setup (please complete the following information if applicable):**
- - OS: [e.g. iOS]
- - IDE : [e.g. IntelliJ]
- - Version of the Library used
+ - Name and Version of the Library package used: [e.g. Azure.Storage.Blobs 12.2.0] 
+ - OS and .NET runtime version: [e.g. Linux dotnet core 3.1]
```
Comment (jongio) [inline .github/ISSUE_TEMPLATE/question-query-template.md]: When we say "clear and concise" above it sounds a bit patronizing.  How about something like "How can we help?"  or "What is your question?"  and assume people will provide the necessary details and if not we can ask follow up questions.

File: .github/ISSUE_TEMPLATE/question-query-template.md [hunk 2]
```diff
@@ -14,9 +14,9 @@ A clear and concise ask/query.
...
+ - IDE and version : [e.g. Visual Studio 16.3]
  
  **Information Checklist**
  Kindly make sure that you have added all the following information above and checkoff the required fields otherwise we will treat the issuer as an incomplete report
```
Comment (jongio) [inline .github/ISSUE_TEMPLATE/question-query-template.md]: I'm not a fan of the tone of this.  Do we need this sentence at all?

File: .github/ISSUE_TEMPLATE/question-query-template.md (hunk 1 above)
Comment (jongio) [reply .github/ISSUE_TEMPLATE/question-query-template.md]: Same goes for the other templates.

**Review by jongio — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/question-query-template.md (hunk 2 above)
Comment (pakrym) [reply .github/ISSUE_TEMPLATE/question-query-template.md]: I agree but would prefer to keep this change targeted to setup section.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -7,6 +7,9 @@ assignees: ''
 
 ---
 
+**Library or service name.**
```
[the remark this answers is not in the document]
Comment (pakrym) [reply .github/ISSUE_TEMPLATE/feature_request.md]: This template doesn't have a setup section.

File: .github/ISSUE_TEMPLATE/question-query-template.md [hunk 4]
```diff
@@ -14,9 +14,9 @@ A clear and concise ask/query.
 A clear explanation of why is this not a bug or a feature request?
 
 **Setup (please complete the following information if applicable):**
- - OS: [e.g. iOS]
- - IDE : [e.g. IntelliJ]
- - Version of the Library used
+ - Name and Version of the Library package used: [e.g. Azure.Storage.Blobs 12.2.0]
```
[the remark this answers is not in the document]
Comment (pakrym) [reply .github/ISSUE_TEMPLATE/question-query-template.md]: Unrelated to the change.

File: .github/ISSUE_TEMPLATE/question-query-template.md (hunk 1 above)
Comment (pakrym) [reply .github/ISSUE_TEMPLATE/question-query-template.md]: Agreed but unrelated to the change.

File: .github/ISSUE_TEMPLATE/question-query-template.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (pakrym) [reply .github/ISSUE_TEMPLATE/question-query-template.md]: Fixed.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 5]
```diff
@@ -7,19 +7,11 @@ assignees: ''
...
+ 
 **Is your feature request related to a problem? Please describe.**
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
-
-**Describe the solution you'd like**
-A clear and concise description of what you want to happen.
+What feature would you like to get added? What problem is it solving?
 
 **Describe alternatives you've considered**
 A clear and concise description of any alternative solutions or features you've considered.
```
Comment (jongio) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Remove clear and concise.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 6]
```diff
@@ -7,19 +7,11 @@ assignees: ''
...
+What library or service is this request related to? [e.g. Azure.Storage.Blobs] 
+ 
 **Is your feature request related to a problem? Please describe.**
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
-
-**Describe the solution you'd like**
-A clear and concise description of what you want to happen.
+What feature would you like to get added? What problem is it solving?
 
 **Describe alternatives you've considered**
```
Comment (jongio) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Actually remove this whole alternatives section

**Review by jongio — APPROVED** [review]

**Pull request merged**